UserConfig.json

```diff
@@ -1,22 +1,29 @@
 {
+    "use_columnar": false,
+    "use_columnar_reduce": false,
+    "use_columnar_rewrite": false,
+    "use_selection_pushdown": false,
```

Comment:
all introduced config options need to be handled in the ConfigParser class. Without that, the config parser will throw an exception if it encounters unknown options.